openwisp_radius/tests/test_api.py

```diff
@@ -208,9 +208,13 @@ def test_api_password_reset(self):
         self.default_org.add_user(test_user)
         mail_count = len(mail.outbox)
         reset_payload = {'email': 'test@email.com'}
+        password_reset_url = reverse('freeradius:rest_password_reset', args=['wrong-org'])
+
+        # no payload
+        response = self.client.post(password_reset_url, data={})
+        self.assertEqual(response.status_code, 400)
```

Comment:
the build is failing because the system correctly returns `404` because `wrong-org` does not exist. I think you need to supply a valid org to trigger the `400`